character-sfx.js

```diff
@@ -55,6 +55,37 @@ const _getActionFrameIndex = (f, frameTimes) => {
   return i;
 };
 
+const jumpGruntIndex={
+  'shishi':[{index:2,nonce:0},{index:3,nonce:0},{index:4,nonce:0},{index:5,nonce:0},{index:6,nonce:0},{index:7,nonce:0}],

```

Comment:
We should not be hard coding any voice actors here.

This must be declared from the voice pack, most likely with a naming scheme.

Reply:
If we really need to, I think it's better to duplicate files in the voice pack.

E.g. copy the same file to `Jump1.wav` so that we can do `/jump/i.test(fileName)`